smartsim/generation/generator.py

```diff
@@ -211,10 +211,25 @@ def _write_tagged_entity_files(self, entity):
         """
         if entity.files:
             to_write = []
-            for tagged_file in entity.files.tagged:
-                dst_path = path.join(entity.path, path.basename(tagged_file))
-                shutil.copyfile(tagged_file, dst_path)
-                to_write.append(dst_path)
+
+            def _build_tagged_files(tagged):
+                """Using a TaggedFileHierarchy, reproduce the tagged file
+                directory structure
+
+                :param tagged: a TaggedFileHierarchy to be built as a
+                               directory structure
+                :type tagged: TaggedFilesHierarchy
+                """
+                for f in tagged.files:
```

Comment:
Avoid single-character variables. Maybe `f` -> `file`?

Reply:
resolved in https://github.com/CrayLabs/SmartSim/pull/119/commits/3cf5b79d4a70cf8019631dea7a0e8641a082ac92